src/scripts.js

```diff
@@ -97,6 +118,69 @@ function getUserById(id, listRepo) {
   return listRepo.getDataFromID(id);
 }
 
+
+function windowOnClick(event) {
```

Comment:
I like this windowOnClick function. I definitely plan to use this in the future!